### PR TITLE
MVC 5 to MVC 6 migration doc fixes

### PR DESCRIPTION
**Pull request by scottaddie**
This PR addresses code snippet errors, inconsistencies, and grammar/punctuation problems. It also attempts to provide clarification where it was needed.

### CODE REVIEW

File: mvc/migration/migratingfrommvc5.rst [hunk 1]
```diff
@@ -225,17 +226,14 @@ View the site in the browser. It should now load correctly, with the expected st
...
+The MVC 5 starter web template utilized the MVC runtime support for bundling. In ASP.NET MVC 6, this functionality is performed as part of the build process using `Gulp <http://gulpjs.com/>`__. We've previously configured bundling and minification; all that's left is to change the references to Bootstrap, jQuery and other assets to use the bundled and minified versions. You can see how this is done in the layout file (*Views/Shared/_Layout.cshtml*) of the full template project. See `Bundling and Minification <http://docs.asp.net/en/latest/client-side/bundling-and-minification.html>`__ for more information.
 
 Additional Resources
 --------------------
 
 - `Migrating an ASP.NET MVC 5 App to ASP.NET 5 <https://weblogs.asp.net/paullitwin/migrating-an-asp-net-mvc-5-app-to-asp-net-5>`__
 - `Using Gulp <http://docs.asp.net/en/latest/client-side/using-gulp.html>`__
-- `Using Grunt with ASP.NET <http://docs.asp.net/en/latest/client-side/using-grunt.html>`_
```
Comment (shirhatti) [inline mvc/migration/migratingfrommvc5.rst]: Why did you remove this link?

**Comment by shirhatti** [comment]
https://github.com/aspnet/Docs/pull/824/files#diff-a6c4e7f724a0eb14b09e27186c8d4ec4L236 and https://github.com/aspnet/Docs/pull/824/files#diff-a6c4e7f724a0eb14b09e27186c8d4ec4L45

If you don't mind updating this PR to use sphinx-style relative links instead of hard links, I'd really appreciate that :smile:

File: mvc/migration/migratingfrommvc5.rst (hunk 1 above)
Comment (scottaddie) [inline mvc/migration/migratingfrommvc5.rst]: @shirhatti It seemed irrelevant to me, since the doc focused on Gulp.

**Comment by scottaddie** [comment]
@shirhatti No problem. I'll get to work on those updates, but it may be a few days because of the Christmas holiday.

**Comment by scottaddie** [comment]
@shirhatti PR updated to use the relative Sphinx-style links. Let me know how it looks.

**Comment by shirhatti** [comment]
Thanks @scottaddie 
I merged your changes in 49c6d4ff00df28d96182fc162aeb2b526a1f0dfe and 2f92cf845b456c7b5359c35144aece1041de6d36
